curator/src/mail_agent/newsletter_txt_format.ts

```diff
@@ -0,0 +1,117 @@
+import { curate } from "../curate";
+
+import { Summary } from "../types";
+
+// links to curate from (TODO : make it based on user preference)
+
+const links = [
+    'https://www.fromjason.xyz/p/notebook/where-have-all-the-websites-gone/',
+    'https://www.joelonsoftware.com/2000/04/06/things-you-should-never-do-part-i/',
+    'https://biomejs.dev/blog/biome-v1-5/',
+    'https://birtles.blog/2024/01/06/weird-things-engineers-believe-about-development/',
+    'https://julesblom.com/writing/flushsync',
+];
+
+// Formats the newsletter in markdown 
+// articles : Summary[]
+// 
+// return newsletter : String
+
+function formatNewsletterMarkdown(articles : Summary[]) {
+
+    // Newletter's header
+
+    let newsletter = 'Newsletter MARKDOWN\n\n';
+    newsletter += "Hello everyone! Here are the latest news!\n\n";
+
+    // Formatting for each article
+
+    articles.forEach(article => {
+        const title = article.title || "Title not available";
+        const author = article.author || "Unknown author";
+        const summary = article.summary || "Summary not available.";
+        const link = article.link || "#";
+
+        
+        newsletter += `### ${title}\n`;
+        newsletter += `*by ${author}*\n`;
+        newsletter += `🔗 [Read the full article](${link})\n\n`;
+        newsletter += `> ${summary}\n\n`;
+    });
+
+    // Footer
+
+    newsletter += "\nThat's all for now! See you soon for more news!\n";
+
+    return newsletter;
+}
+
+// Formats the newsletter in html with style
+// articles : Summary[]
+// 
+// return newsletter : String
+
+function formatNewsletterHtmlWithCSS(articles: Summary[]) {
```

Comment:
**suggestion:** Maybe you can use the markdown output to format the HTML through the (e.g. https://github.com/showdownjs/showdown)

Reply:
The markdown version is just considered as an "alt" version for the mail. The HTML version is closer to the result we want to achieve in the final product. So basing the HTML version off the markdown would probably not match our needs.